.travis.yml

```diff
@@ -25,6 +25,11 @@ matrix:
 
     - name: "j"
       language: java
+      addons:
+        apt:
+          packages:
+            - ant
+            - ant-optional
```

Comment:
What is ant-optional and do we need it?  (minor issue)

Reply:
Turns out that we do need it. :)
I first tried with just `ant` and while that did unblock the ICU4J builds, they eventually failed later on because the Ant JUnit plugins/extensions weren't installed.
They are part of the Ant Optional package, so it seems that we do need it.
(Not sure if that might be worth adding a comment to the YML file for or not though...?)